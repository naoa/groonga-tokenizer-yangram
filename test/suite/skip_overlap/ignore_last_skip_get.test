register tokenizers/yangram

tokenize TokenYaBigram "今日は雨だ" NormalizerAuto --mode GET
