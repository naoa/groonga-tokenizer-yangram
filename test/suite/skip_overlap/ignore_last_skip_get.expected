register tokenizers/yangram
[[0,0.0,0.0],true]
tokenize TokenYaBigram "今日は雨だ" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0
    },
    {
      "value": "は雨",
      "position": 2
    },
    {
      "value": "雨だ",
      "position": 3
    }
  ]
]
